### PR TITLE
Upgrade security-policy datasource - part 9

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves provider upgrade

Test passes on fork: https://github.com/vmanilo/terraform-provider-twingate/actions/runs/5835678633

## Changes
- upgrade security-policy datasource

### CODE REVIEW

File: docs/data-sources/security_policies.md [hunk 1]
```diff
@@ -19,15 +19,18 @@ data "twingate_security_policies" "all" {}
...
 
+### Optional
+
+- `security_policies` (Attributes List) (see [below for nested schema](#nestedatt--security_policies))
+
 ### Read-Only
 
-- `id` (String) The ID of this resource.
-- `security_policies` (Block List) (see [below for nested schema](#nestedblock--security_policies))
+- `id` (String) The ID of the data source result
```
Comment (vmanilo) [inline docs/data-sources/security_policies.md:28]: @ekampf should I revert this changes?

File: docs/data-sources/security_policies.md [hunk 2]
```diff
@@ -19,15 +19,18 @@ data "twingate_security_policies" "all" {}
...
 
-<a id="nestedblock--security_policies"></a>
+<a id="nestedatt--security_policies"></a>
 ### Nested Schema for `security_policies`
 
 Read-Only:
 
-- `id` (String) Return a matching Security Policy by its ID. The ID for the Security Policy can be obtained from the Admin API or the URL string in the Admin Console.
+- `id` (String) The ID of the Security Policy resource.
```
Comment (vmanilo) [inline docs/data-sources/security_policies.md:35]: @ekampf should I revert this changes?

**Review by ekampf — APPROVED** [review]
